blocks/faq/src/question.tsx

```diff
@@ -0,0 +1,192 @@
+  question?: string;
+  answer?: string;
+  deletable?: boolean;
+  readonly?: boolean;
+  displayNumber?: boolean;
+  displayToggle?: boolean;
+  updateField: (value: string, field: QuestionOrAnswer) => void;
+  onRemove: () => void;
+}
+
+export const Question: FunctionComponent<QuestionProps> = ({
+  index,
+  question,
+  answer,
+  deletable = true,
+  readonly,
+  displayNumber = true,
+  displayToggle = true,
+  updateField,
+  onRemove,
+}) => {
+  const [questionValue, setQuestionValue] = useState(question ?? "");
+  const [answerValue, setAnswerValue] = useState(answer ?? "");
+  const [expanded, setExpanded] = useState(true);
+
+  return (
+    <Box display="flex" alignItems="flex-start">
+      <Box>
+        <Button
+          variant="tertiary_quiet"
+          onClick={() => setExpanded(!expanded)}
+          sx={{
+            minWidth: 0,
+            minHeight: 0,
+            paddingX: 0,
+            paddingY: readonly ? 0 : 2.125,
+            mr: displayNumber || displayToggle ? 1.5 : 0,
+            background: "none !important",
+            fontSize: 12,
+            fontWeight: 700,
+            lineHeight: "15px",
+            color: ({ palette }) => palette.black,
+            transition: ({ transitions }) => transitions.create("margin-right"),
+          }}
+        >
+          <Collapse in={displayNumber} orientation="horizontal">
+            {index}
+          </Collapse>
+          <Collapse in={displayToggle} orientation="horizontal">
+            <CaretDownIcon
+              sx={{
+                fontSize: 12,
+                color: ({ palette }) => palette.black,
+                transition: ({ transitions }) =>
+                  transitions.create(["transform", "margin-left"]),
+                transform: `rotate(${expanded ? 0 : -90}deg)`,
+                ml: displayNumber ? 0.75 : 0,
+              }}
+            />
+          </Collapse>
+        </Button>
+      </Box>
+
+      <Box
+        sx={{
+          flexGrow: 1,
+          ...(!readonly
+            ? {
+                border: ({ palette }) => `1px solid ${palette.gray[20]}`,
+                borderRadius: 2.5,
+                overflow: "hidden",
+              }
+            : {}),
+        }}
+      >
+        <Box
+          sx={{
+            ...(!readonly
+              ? {
+                  display: "flex",
+                  justifyContent: "space-between",
+                  alignItems: "center",
+                  width: 1,
+                  paddingY: 2.125,
+                  paddingX: 2.75,
+                }
+              : {}),
+          }}
+        >
+          <EditableField
+            editIconFontSize={15}
+            value={questionValue}
+            onChange={(event) => {
+              if (!readonly) {
+                setQuestionValue(event.target.value);
+              }
+            }}
+            onBlur={(event) => updateField(event.target.value, questionKey)}
+            sx={{
+              width: 1,
```

Comment:
On the default style of `EditableField`, it already has `width: 1` if I'm not mistaken. We can remove this if that's the case (please confirm before doing so)